settings/request_test.go

```diff
@@ -27,13 +28,26 @@ func (*ValidSchemeResolver) ResolveSpaceIndex(s, i interface{}) (uint32, uint32,
 	} else {
 		spaceNo = 0
 	}
+
+	return spaceNo, nil
+}
+
+func (*ValidSchemeResolver) ResolveIndex(i interface{}, spaceNo uint32) (uint32, error) {
+	var indexNo uint32
 	if i != nil {
 		indexNo = uint32(i.(int))
 	} else {
 		indexNo = 0
 	}
+	return indexNo, nil
+}
 
-	return spaceNo, indexNo, nil
+func (*ValidSchemeResolver) NamesUseSupported() bool {
+	isLess, err := test_helpers.IsTarantoolVersionLess(3, 0, 0)
```

Comment:
The same as for `crud/requests_test.go`.

Reply:
Updated.